### ISSUE TITLE
add an api for changing statuses

### ISSUE THREAD

**Issue by williamhorning** [jeffalo/my-ocular#5]
**Is your feature request related to a problem? Please describe.**
Yes. My project (1Status) wants to be able to change ocular statuses, but we can't.
**Describe the solution you'd like**
Add an api for this using tokens that jeffalo can give out.
**Describe alternatives you've considered**
Directing users to ocular
**Additional context**
![image](https://user-images.githubusercontent.com/68466727/113490674-8fe52b80-9499-11eb-831c-0e743c2513cb.png)

**Comment by jeffalo** [comment]
yeah. im working on some kind of system with api tokens and stuff. one day!

**Comment by pikachub2005** [comment]
...
